content/docs/multi-user/overview.md

```diff
@@ -0,0 +1,116 @@
++++
+title = "Overview of Multi-user Isolation Support"
+description = "Isolation of user created resources for convenience and organization"
+weight = 5
++++
+
+Kubeflow v0.6 release enables support for multi-user isolation of user created
+resources in a Kubeflow deployment. The primary purpose of this functionality
+is to enable multiple users to operate on a shared Kubeflow deployment without
+stepping on each others' jobs and resources. It provides the users with the
+convenience of clutter free browsing of notebooks, training jobs, serving
+deployments and other resources. The isolation mechanisms also prevent
+accidental deletion/modification of resources of other users in the deployment.
+Note that the isolation support in Kubeflow doesn't provide any hard security
+guarantees against malicious attempts by users to infiltrate other user's
+profiles.
+
+## Usage overview
+
+An administrator needs to deploy Kubeflow and configure the authentication.
+service for the deployment.  A user can log into the system and will by default
+be accessing their primary profile. Users have view and modify access to their
+primary profile. The owner of a profile can share access to the profile with
+another user in the system. When sharing the access to a profile, the owner can
+choose to share it as a view access only or view and modify access with the
+user.
+
+A user can select their active profile from the top bar on
+the Kubeflow Central Dashboard. Note that the user can only view the profiles
+in the dropdown list that they have view or modify access to.
+
+Todo(abhishek): add screenshot
+
+For this overview we will illustrate the user isolation functionality using the
+Jupyter notebooks service which is the first service in the system to have full
+integration with the multi-user isolation functionality.  
+
+Once an active profile has been selected by the user, the Notebooks Manager UI
```

Comment:
Should we use `active namespace` or `active profile`?
The bar on central dashboard now still use namespace.

Reply:
Good point. I'd prefer to use the same terminology everywhere. @avdaredevil Can you please confirm? Also, waiting on the dashboard update to clarify this.